src/components/blocks/software/__snapshots__/Pre.test.tsx.snap

```diff
@@ -1,113 +1,35 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`<Pre /> should render code block when no languages are passed 1`] = `
-<div
-  class="my-32"
+<pre
```

Comment:
Please can you run `npm run repo-githooks` so you can run tests before commits? This will make snapshot tests catch changes sooner.